fix(docker): Agregar module-revision-tela al Dockerfile

## Descripción del Problema

El build de Docker falla porque el módulo `module-revision-tela` no está incluido en el Dockerfile.

**Error:**
```
[ERROR] Child module /app/module-revision-tela of /app/pom.xml does not exist
```

## Causa

El Dockerfile no incluye las instrucciones COPY para:
- `module-revision-tela/pom.xml`
- `module-revision-tela/src`

## Solución

Agregar las líneas correspondientes en el Dockerfile para copiar el pom.xml y el código fuente del módulo.

--- a/Dockerfile
+++ b/Dockerfile
@@ -28,6 +28,7 @@ COPY ./module-websocket/pom.xml ./module-websocket/pom.xml
 COPY ./module-driver/pom.xml ./module-driver/pom.xml
 COPY ./module-comercial/pom.xml ./module-comercial/pom.xml
 COPY ./module-liquidaciones/pom.xml ./module-liquidaciones/pom.xml
+COPY ./module-revision-tela/pom.xml ./module-revision-tela/pom.xml
 
 # Descargar dependencias (aprovechando caché de Docker)
 RUN mvn dependency:go-offline -B
@@ -48,6 +49,7 @@ COPY ./module-websocket/src ./module-websocket/src
 COPY ./module-driver/src ./module-driver/src
 COPY ./module-comercial/src ./module-comercial/src
 COPY ./module-liquidaciones/src ./module-liquidaciones/src
+COPY ./module-revision-tela/src ./module-revision-tela/src
 COPY ./src ./src
 
 # Compilar el proyecto completo (avro-schemas ya está disponible en .m2)

